src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private static final String name = "circle";
```

Comment:
```suggestion
```
You don't need this field. Remove it from all the pieces. You can just write "circle" in toString